packages/typespec-python/test/generated/internal/internal/_operations/_operations.py

```diff
@@ -0,0 +1,267 @@
+
+
+def build_internal_post_internal_request(**kwargs: Any) -> HttpRequest:
+    _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+
+    content_type: Optional[str] = kwargs.pop("content_type", _headers.pop("Content-Type", None))
+    accept = _headers.pop("Accept", "application/json")
+
+    # Construct URL
+    _url = "/internal/postInternal"
+
+    # Construct headers
+    _headers["Accept"] = _SERIALIZER.header("accept", accept, "str")
+    if content_type is not None:
+        _headers["Content-Type"] = _SERIALIZER.header("content_type", content_type, "str")
+
+    return HttpRequest(method="POST", url=_url, headers=_headers, **kwargs)
+
+
+class InternalClientOperationsMixin(InternalClientMixinABC):
+    @distributed_trace
+    def _get_internal(self, *, name: str, **kwargs: Any) -> _models._models.InternalModel:
+        """get_internal.
+
+        :keyword name: Required.
+        :paramtype name: str
+        :keyword bool stream: Whether to stream the response of this operation. Defaults to False. You
+         will have to context manage the returned stream.
+        :return: InternalModel. The InternalModel is compatible with MutableMapping
+        :rtype: ~internal.models.InternalModel
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+        error_map = {
+            401: ClientAuthenticationError,
+            404: ResourceNotFoundError,
+            409: ResourceExistsError,
+            304: ResourceNotModifiedError,
+        }
+        error_map.update(kwargs.pop("error_map", {}) or {})
+
+        _headers = kwargs.pop("headers", {}) or {}
+        _params = kwargs.pop("params", {}) or {}
+
+        cls: ClsType[_models._models.InternalModel] = kwargs.pop("cls", None)  # pylint: disable=protected-access
+
+        request = build_internal_get_internal_request(
+            name=name,
+            headers=_headers,
+            params=_params,
+        )
+        request.url = self._client.format_url(request.url)
+
+        _stream = kwargs.pop("stream", False)
+        pipeline_response: PipelineResponse = self._client._pipeline.run(  # pylint: disable=protected-access
+            request, stream=_stream, **kwargs
+        )
+
+        response = pipeline_response.http_response
+
+        if response.status_code not in [200]:
+            map_error(status_code=response.status_code, response=response, error_map=error_map)
+            raise HttpResponseError(response=response)
+
+        if _stream:
+            deserialized = response.iter_bytes()
+        else:
+            deserialized = _deserialize(
+                _models._models.InternalModel, response.json()  # pylint: disable=protected-access
+            )
+
+        if cls:
+            return cls(pipeline_response, deserialized, {})  # type: ignore
+
+        return deserialized  # type: ignore
+
+    @overload
+    def _post_internal(
```

Comment:
i feel we don't need to generate overloads for internal methods. Thought @msyyc @johanste @Azure/arm-template-deployments

Reply:
ok talked with @lmazuel: let's do the `typing.overload`s, but not generate the overload specific docstrings. That way we still have intellisense for sdk authors, but we don't generate a lot of unnecessary doc code.

Reply:
shall we also need to remove the doc code for internal operation?